src/comments/collectCommentFileNames.ts

```diff
@@ -26,12 +28,27 @@ export const collectCommentFileNames = async (
             );
         }
 
+        const includeList = uniqueFromSources(
+            typescriptConfiguration.files,
+            typescriptConfiguration.include,
+        );
+
+        // Remove directories specified in the include list. Ignore
+        // Errors as they'll be handled when reading the file.
+        const includeListFiles = (
+            await Promise.all(includeList.map(dependencies.fileSystem.directoryExists))
+        )
+            .map((isDirectory, i) => {
+                if (isDirectory) {
+                    return null;
+                }
+                return includeList[i];
+            })
+            .filter((item): item is string => typeof item === "string");
```

Comment:
Nit: there's a util for this already:

```suggestion
            .filter(isTruthy);
```